README.md

```diff
@@ -864,14 +864,14 @@ Consider this small, contrived application:
 
         # Example 1 : exception thrown when name has non-printable character
         try:
-            create_notes( session, "lucky\033.dat", content = u'test' )
+            create_notes( session, "lucky\1.dat", content = u'test' )
```

Comment:
Why replace 033 with 1?

Reply:
Really just a practical matter , ils was rending it as `luckyat` which was confusing. If you did `iquest "select DATA_NAME where DATA_NAME like 'luck%'"|od -bc you saw the escape (\033)and the following two characters, though.
'x\01' is maybe a better demo since it also produces the XML error (without special parser being set) but doesn't mangle the data object name in your terminal.

Reply:
Got it.